### DISCUSSION TITLE
Max Context Length of a model

### DISCUSSION THREAD

**Discussion by rounak610** [ggml-org/llama.cpp#3474] [Q&A]
Is there any way I can find the maximum context length of a local LLM?

**Comment by staviq** [comment]
Besides checking the model page,

If you run `main`, it tells you what is the context length the model was trained on ( basically the intended max context length )

For example, with mistral-openorca you'll see this in the console output:

```
llm_load_print_meta: n_ctx_train      = 32768
```

Meaning, the intended max context length is 32k in this case.
